cime_config/tests.py

```diff
@@ -436,7 +436,6 @@
 
     "e3sm_gpuomp" : {
         "tests"    : (
-                 "SMS_Ln9.ne4pg2_ne4pg2.F-MMFOMP.eam-single_thread",
```

Comment:
@xyuan as discussed on Slack

Reply:
Why is it being removed?

Reply:
It's been broken since Feb 7 with no plans to fix. When fixed, this can be reverted.